### PR TITLE
Reactivate CORS

### PR DESCRIPTION
**Pull request by mmacata**
To get rid of nginx for deployment, the openeo-grassgis-driver needs to handle it by itself. This PR reactivates it.

Related to https://github.com/Open-EO/openeo-grassgis-driver/pull/85 and https://github.com/Open-EO/openeo-grassgis-driver/pull/86

It was sucessfully tested locally with the openeo webeditor.

### CODE REVIEW

File: src/openeo_grass_gis_driver/app.py [hunk 1]
```diff
@@ -11,7 +11,6 @@
 
 flask_app = Flask(__name__)
 flask_app.wsgi_app = ProxyFix(flask_app.wsgi_app, x_for=1, x_host=1, x_proto=1)
-# as flask CORS lead to unforeseen behaviour in certain cases, we configure
```
Comment (metzm) [inline src/openeo_grass_gis_driver/app.py:14]: Have you tested these "certain cases"?

Comment (mmacata) [reply src/openeo_grass_gis_driver/app.py:14]: The certain cases do not apply any more, see second part of https://github.com/Open-EO/openeo-grassgis-driver/pull/85#issuecomment-829236931.

**Review by marcjansen — APPROVED** [review]
Nice

**Review by metzm — APPROVED** [review]
Ready to merge.

**Pull request merged**